Form-Controls/index.html

```diff
@@ -7,21 +7,63 @@
   <meta name="description" content="">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" href="styles.css">
+    <!-- write your html here-->
 </head>
 <body>
   <header>
     <h1>Product Pick</h1>
+
+    <div class="contact form">
+    <form >
+    <label for="Name">Your Name</label>
+    <input type="text" placeholder="Name">
+
+<label for="email"> Your Email</label>
```

Comment:
When you tested your form, what happened when you submitted an invalid email? 

What about no email?